.github/workflows/pnp-build-input-api.yml

```diff
@@ -209,6 +214,13 @@ jobs:
         with:
           service-account-key: ${{ secrets.GCLOUD_AUTH_STAGING }}
 
+      - name: Restore AsyncAPI schema from cache
```

Comment:
What do we mean with cache here? I am asking since I do not know the action. Is it really just extracting it from the build cache?